src/sockaddr.rs

```diff
@@ -305,6 +305,15 @@ impl fmt::Debug for SockAddr {
     }
 }
 
+impl clone::Clone for SockAddr {
```

Comment:
I think we can just derive using `#[derive(Clone)]`.

Reply:
Right--replaced that :+1: